lib/SearchAndSort/tests/advancedSearchQueryToRows-test.js

```diff
@@ -38,4 +38,17 @@ describe('advancedSearchQueryToRows', () => {
       });
     });
   });
+
+  describe('when query without match and search option', () => {
+    it('should parse query correctly', () => {

```

Comment:
Could you please clarify should we rename to `should use default value for match and search option` or something like that?